docs/index.html

```diff
@@ -1,79 +1,112 @@
-<!doctype html>
-<html lang="en">
-    <head>
-        <meta charset="utf-8">
-        <title>harp.gl</title>
-        <meta name="description" content="Landing page for harp.gl">
-        <meta name="viewport" content="width=device-width">
-        <link rel="stylesheet" href="./css/style.css">
-        <script   src="https://code.jquery.com/jquery-3.4.1.min.js" integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
+<html>
+   <head>
+      <title>harp.gl</title>
+      <link href="https://fonts.googleapis.com/css?family=Fira+Sans:400,500,600&display=swap" rel="stylesheet">
+      <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+      <meta name="viewport" content="width=device-width,initial-scale=1">
+      <link rel="stylesheet" type="text/css" href="css/index.css">
+      <link rel="shortcut icon" href="img/favicon.ico">
+   </head>
+   <body>
+      <header>
+         <iframe src="https://dbabbs.github.io/harp-temp-site/dist/"></iframe>
 
-        <link rel="icon" type="image/x-icon" href="img/favicon.ico">
-
-    </head>
-    <body class="here">
-        <header class="panel">
-            <div class="content-wrapper">
-                <nav id="service_switcher">
-                    <ul>
-                        <li>
-                            <a class="logo" href="https://here.com" title="Go to HERE.com to find out more about the company"></a>
-                        </li>
-                    </ul>
-                </nav>
-                <div class="header-descr">
-                    <h1 class="title">harp.gl</h1>
-                    <p class="intro">3D Web Rendering Engine.</p>
-                    <div>
-                            <a href="https://github.com/heremaps/harp.gl" class="dp-button dp-button-transparent" target="_blank">BROWSE ON GITHUB</a>
-                    </div>
-                </div>
+         <div class="container header-container">
+            <nav>
+               <div class="logo"></div>
+               <div>
+                  <a id="docs-nav-mobile" href="http://harp.gl/docs" class="nav-item">Docs</a>
+                  <a id="docs-nav" href="http://harp.gl/docs" class="nav-item">Documentation</a>
+                  <a href="http://harp.gl/examples" class="nav-item">Examples</a>
```

Comment:
This goes to a different page, when compared with the examples button lower in the page, is this intended, my gut feeling is that this is not desired.

Reply:
That is by design. Currently this link (http://harp.gl/docs) doesn't work because the site hasn't been published. However, if you view @musculman's forked published version, you can see the short url working:

https://musculman.github.io/harp.gl/docs
https://musculman.github.io/harp.gl/examples

During jsconf asia, we identified the need for short urls to the examples and docs, rather than the long s3 bucket urls.

Reply:
Ok, thanks for clarifying.